config/manager/manager.yaml

```diff
@@ -71,6 +71,8 @@ spec:
               value: quay.io/eclipse/che-jwtproxy:0.10.0
             - name: RELATED_IMAGE_single_host_gateway
               value: quay.io/eclipse/che--traefik:v2.3.2-6e6d4dc5a19afe06778ca092cdbbb98e31cb9f9c313edafa23f81a0e6ddf8a23
+            - name: RELATED_IMAGE_single_host_gateway_native_user_mode
```

Comment:
We've already moved to `quay.io/eclipse/che--traefik:v2.5.0-rc2-df90799aaca1ad6fb9e06d311140035d2a0c2295a4f8f508f6b55ee056bb677e` for `RELATED_IMAGE_single_host_gateway` Can we reuse this env var without introducing a new one ?

Reply:
reverted at https://github.com/eclipse-che/che-operator/pull/952 so unfortunately we would temporary need this. Alternatively, I can hardcode it in code so it's not extra env variable in deployment. wdyt @tolusha ? It will be removed once stable 2.5 traefik is released.